src/WorkflowsRules/Services/Actions.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Kanvas\Packages\WorkflowsRules\Services;
+
+use Kanvas\Packages\WorkflowsRules\Actions\ADF;
+
+use Kanvas\Packages\WorkflowsRules\Actions\PDF;
+use Kanvas\Packages\WorkflowsRules\Actions\SendMail;
+use Kanvas\Packages\WorkflowsRules\Actions\SendToZoho;
+use Kanvas\Packages\WorkflowsRules\Contracts\Interfaces\ActionInterfaces;
+
+use Kanvas\Packages\WorkflowsRules\Models\Rules;
+
+class Actions
+{
+
+    /**
+     * getAction.
+     *
+     * @param  mixed $action
+     * @param  mixed $rule
+     *
+     * @return ActionInterfaces
+     */
+    public static function getAction(string $actionClass, Rules $rule, $log) : ActionInterfaces
```

Comment:
@FredPeal shouldnt log have a type?

and add todo for in furture avoid it being hardcoded